cstar_ocean/input_dataset.py

```diff
@@ -91,35 +98,59 @@ def __str__(self):
     def __repr__(self):
         return self.__str__()
 
-    def get(self, local_path: str):
+    def get(self, local_dir: str):
         """
-        Fetch the file containing this input dataset and save it to `local_path` using Pooch.
+        Make the file containing this input dataset available in `local_dir/input_datasets`
 
-        This method updates the `local_path` attribute of the calling InputDataset object
+        If InputDataset.source is...
+           - ...a local path: create a symbolic link to the file in `local_dir/input_datasets`.
+           - ...a URL: fetch the file to `local_dir/input_datasets` using Pooch
+                       (updating the `local_path` attribute of the calling InputDataset)
```

Comment:
Yeah this is another argument for abstracting out the concept of a "source" into a standalone class / type. 

This is a similar idea to using `cloudpathlib`, but maybe we need to attach more info, in which case we would need our own `Source` class with a `.path` attribute.